apps/dolly-frontend/src/main/js/src/components/fagsystem/sigrunstubSummertSkattegrunnlag/form/KjoeretoeyArrayForm.tsx

```diff
@@ -0,0 +1,65 @@
+import { getInitialKjoeretoey } from '@/components/fagsystem/sigrunstubSummertSkattegrunnlag/form/Form'
+import { FormDollyFieldArray } from '@/components/ui/form/fieldArray/DollyFieldArray'
+import { FormSelect } from '@/components/ui/form/inputs/select/Select'
+import { FormTextInput } from '@/components/ui/form/inputs/textInput/TextInput'
+
+export const KjoeretoeyArrayForm = ({ path }) => (
+	<FormDollyFieldArray
+		name={path}
+		header="Spesifisering"
+		newEntry={getInitialKjoeretoey()}
+		canBeEmpty={true}
+		nested
+	>
+		{(kjoeretoeyPath, idx) => (
+			<div className="flexbox--flex-wrap sigrun-form" key={idx}>
+				<FormSelect
+					name={`${kjoeretoeyPath}.type`}
+					info={'Foreløpig er det kun støtte for kjøretøy'}
```

Comment:
Funker ikke så bra å ha den her, infoknappen blir veldig liten, og hele feltet blir forskjøvet nedover. Ville hatt den som en egen hjelpetekst til høyre for feltet. Ser vi f.eks. har gjort det slik andre steder:
`<div
														style={{ marginLeft: '-20px', marginRight: '20px', paddingTop: '5px' }}
													>
														<Hjelpetekst>
															Foreløpig er det kun støtte for kjøretøy.
														</Hjelpetekst>
													</div>`